### PR TITLE
Adding public trello workflow

### PR DESCRIPTION
**Pull request by hflynn**
Updating https://www.openmicroscopy.org/site/support/contributing/team-communication.html#trello and https://www.openmicroscopy.org/site/support/contributing/team-workflow.html with an attempt to describe our new Trello process. also updated Dev Contact list link to new openmicroscopy.org link & added Zoom to voice contact methods.

### CODE REVIEW

File: contributing/team-communication.txt [hunk 1]
```diff
@@ -84,7 +84,9 @@ continuous integration tools.
 
 You can request access to the openmicroscopy boards as an external
 collaborator. Sign up for a free account and then get in touch with us to be
-added.
+added. We have now added a `public OME organisation <https://trello.com/ome>`_
```
Comment (mtbc) [inline contributing/team-communication.txt]: organization

File: contributing/team-workflow.txt [hunk 2]
```diff
@@ -6,38 +6,41 @@ Development management
...
+for higher level planning and will be of most interest to those concerned with
+new features and functionality (Trac is still used for individual bug
+tracking). The 'Getting Started' board provides full instructions and an index
+of current boards, allowing you to browse upcoming work for the next releases
+and topic-based 'epic' work plans.
+
+.. figure:: /images/public_trello.png
+   :align:  center
+
+You can sign up for a free Trello account to add comments and need to be added
```
Comment (mtbc) [inline contributing/team-workflow.txt]: maybe "... and you need to ..."

**Comment by pwalczysko** [comment]
Otherwise reads fine. Links work.

**Comment by joshmoore** [comment]
As mentioned in Jabber, happy with this, though it's likely to develop over the next few weeks. :thumbsup:

**Pull request merged**